### ISSUE TITLE
Add more fields from Hastus

### ISSUE THREAD

**Issue by sherson** [umts/incidents#207]
This came up while I was at SATCo:

Jamin would like to add more fields to Incidents (that are in Hastus). They are recently-created fields, which is why they’re not in there now. I asked them to get us a list. 

On a tangential subject, Missy asked about which fields populate when it's imported into Hastus, and if we can pre-fill-out the Hastus F12 checkboxes during the import. Missy said it might be a question for Laurel.

**Comment by dfaulken** [comment]
I don't believe we've received an update on this from Jamin or others. Please correct me if I'm mistaken.

Yes, seeing as we have not yet been able to test any HASTUS imports, this would be a theoretical question that Laurel would arguably be best poised to answer.

**Comment by sherson** [comment]
I've not heard anything either. Will add it to the list of things to ask Laurel and/or Jonathan about.
